src/map/handler/Map.TouchZoom.js

```diff
@@ -109,6 +109,15 @@ L.Map.TouchZoom = L.Handler.extend({
 		    .off(document, 'touchmove', this._onTouchMove)
 		    .off(document, 'touchend', this._onTouchEnd);
 
+		var snap = L.Browser.any3d ? this._map.options.zoomSnap : 1;
+		if (snap) {
```

Comment:
There's still no need to change `L.Browser.chrome`. Just check for edge first, then chrome, then ffx, then everything else.

Either remove this PR and create a new one, or amend it to remove the unneeded changes. That's up to you.